fastlane/lib/fastlane/actions/git_branch.rb

```diff
@@ -16,7 +16,7 @@ def self.description
       end
 
       def self.details
-        "If no branch could be found, this action will return an empty string. This is a wrapper for the internal action Actions.git_branch"
+        "If no branch could be found, this action will return an empty string. If `FL_GIT_BRANCH_DONT_USE_ENV_VARS` is `true`, it'll ignore CI ENV vars. This is a wrapper for the internal action Actions.git_branch"
```

Comment:
Instead of just an environment variable, can this `dont_use_env_vars` be added as on option on the `git_branch` action?

The action would then pass this as an optional parameter into the `git_branch` method?

I'm kind of leaning towards this so that this behavior is more publicly document in the action options documentation 🤔 Thoughts?

Reply:
The problem here is that git_branch is not the only action relying on that `Helper.git_branch`.

That's the whole problem and why we thought about the env car as being able to impact all actions relying on that helper.

For example, `ensure_git_branch` action is also impacted by that helper's behavior (see PR description for details)